src/ast.rs

```diff
@@ -366,28 +375,42 @@ impl PartialOrd for RawBinaryOperator {
     }
 }
 
+/// This serves a similar purpose to ParenthesizedExpr.
+///
+/// see <https://google.github.io/xls/dslx_reference/#block-expressions>
+pub struct BlockExpr(pub Expr);
+
 /// *Part* of a let binding: the variable declaration and value it is bound to. What is missing
 /// is the expression in which the bound name is in-scope.
+///
+/// see <https://google.github.io/xls/dslx_reference/#let-expression>
 #[derive(Debug, PartialEq, Clone)]
 pub struct RawLetBinding {
     pub variable_declaration: BindingDecl,
-    pub value: Box<Expression>,
+    pub value: Box<Expr>,
```

Comment:
Sorry; "Expression" was fine. Feel free to go whichever route you prefer. I like both just fine. Earlier Rob was a jerk